src/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -18,18 +18,27 @@ import("${chip_root}/src/platform/device.gni")
 
 import("${chip_root}/build/chip/buildconfig_header.gni")
 import("${chip_root}/src/crypto/crypto.gni")
+import("${chip_root}/third_party/silabs/silabs_board.gni")
 
 silabs_platform_dir = "${chip_root}/src/platform/silabs"
 
 assert(chip_device_platform == "SiWx917")
-
-if (chip_enable_openthread) {
-  import("//build_overrides/openthread.gni")
-}
-
 if (chip_crypto == "platform") {
   import("//build_overrides/mbedtls.gni")
 }
+
+config("efr32-platform-wifi-config") {
+  include_dirs = [ "wifi" ]
+  defines = []
+
+  if (use_rs9116 && use_rs911x_sockets) {
```

Comment:
shouldn't this be use_siwx917

Reply:
add on to chirags question, why is there and if (917) when its a 917 build.gn?

Reply:
done 

Only used one condition use_rs911x_sockets